src/lib.rs

```diff
@@ -312,12 +312,13 @@ impl Client {
 
     // Attempts to connect to faktory with exponential backoff + jitter. Largest sleep is 32 seconds
     async fn attempt_connect(config: &Config) -> Option<Connection> {
-        let mut backoff = 1;
+        let mut backoff = 0;
         loop {
-            let jitter = thread_rng().gen_range(1..=backoff);
+            let backoff_ms = (1 << backoff) * 1000;
+            let jitter = thread_rng().gen_range(1..=backoff_ms);
```

Comment:
Will this work? I thought 1..=0 would be empty, that's why I set the initial backoff at 1.

But I really like using ms to give more granularity

Reply:
We'll do `(1 << 0) * 1000` which is just 1000 `(1 << 0 == 1)`. So the smallest range will be `0..=1000`

Reply:
Oh awesome, I missed that!